build.yaml

```diff
@@ -436,7 +436,7 @@ steps:
      fi
      # created ci
      # create test
-     N=$(kubectl -n {{ default_ns.name }} get secret --ignore-not-found=true --no-headers test-tokens | wc -l | tr -d '[:space:]')
+     N=$(kubectl -n {{ batch_pods_ns.name }} get secret --ignore-not-found=true --no-headers test-tokens | wc -l | tr -d '[:space:]')
```

Comment:
Can you get rid of these two changes with `batch_pods_ns.name`. It won't block your PR and I wanted Dan to look at it first in my PR.